packages/build-next-bricks/src/buildStories.js

```diff
@@ -0,0 +1,90 @@
+import webpack from "webpack";
+import path from "path";
+import { createRequire } from "module";
+import { existsSync } from "node:fs";
+import { writeFile, rm } from "node:fs/promises";
+
+const require = createRequire(import.meta.url);
+
+function getBuildStoriesWebpackConfig(packageDir, storiesPath) {
+  const outputPath = path.join(packageDir, "dist");
+
+  return {
+    entry: storiesPath,
+    output: {
+      publicPath: `bricks/${path.basename(packageDir)}/dist/`,
+      path: outputPath,
+      filename: `stories.cjs`,
+      library: {
+        type: "commonjs",
+        export: "default",
+      },
+      clean: true,
+    },
+    resolve: {
+      extensions: [".ts", ".tsx", ".js", ".jsx", ".json"],
+      extensionAlias: {
+        ".js": [".ts", ".tsx", ".js", ".jsx"],
+      },
+    },
+    module: {
+      rules: [
+        {
+          test: /\.ts?$/,
+          loader: "babel-loader",
+          exclude: /node_modules/,
+        },
+        {
+          test: /\.svg$/i,
+          type: "asset/resource",
+          generator: {
+            filename: "stories-asserts/[name].[contenthash][ext]",
```

Comment:
注意 `stories-asserts` 是 jo 之前的 typo，新的改过来 `stories-assets`
```suggestion
            filename: "stories-assets/[name].[contenthash][ext]",
```